x-pack/plugin/security/qa/multi-cluster/build.gradle

```diff
@@ -6,46 +6,76 @@
  */
 
 import org.elasticsearch.gradle.Version
-import org.elasticsearch.gradle.internal.info.BuildParams
 import org.elasticsearch.gradle.internal.test.RestIntegTestTask
 
-
 apply plugin: 'elasticsearch.standalone-rest-test'
 apply plugin: 'elasticsearch.rest-resources'
 
-// randomise between sniff and proxy modes
-boolean proxyMode = (new Random(Long.parseUnsignedLong(BuildParams.testSeed.tokenize(':').get(0), 16))).nextBoolean()
+configurations {
+  signedCerts
+  rootCert
+}
+
+dependencies {
+  signedCerts project(path: ':x-pack:plugin:core', configuration: 'signedCerts')
+  rootCert project(path: ':x-pack:plugin:core', configuration: 'rootCert')
+}
+
+tasks.register("copyCerts", Sync) {
+  dependsOn configurations.signedCerts
+  from(configurations.signedCerts)
+  from(configurations.rootCert)
+  into "${buildDir}/certs"
+}
+
 
 def fulfillingCluster = testClusters.register('fulfilling-cluster') {
   requiresFeature 'es.untrusted_remote_cluster_feature_flag_registered', Version.fromString("8.5.0")
   setting 'xpack.license.self_generated.type', 'trial'
   setting 'remote_cluster.enabled', 'true'
   setting 'remote_cluster.port', '0'
+
+  extraConfigFile 'transport.key', file("${buildDir}/certs/n1.c1.key")
+  extraConfigFile 'transport.cert', file("${buildDir}/certs/n1.c1.crt")
+  extraConfigFile 'transport.ca', file("${buildDir}/certs/ca.crt")
+
+  setting 'xpack.security.transport.ssl.enabled', 'true'
+  setting 'xpack.security.transport.ssl.key', 'transport.key'
+  setting 'xpack.security.transport.ssl.certificate', 'transport.cert'
+  setting 'xpack.security.transport.ssl.certificate_authorities', 'transport.ca'
+  setting 'xpack.security.transport.ssl.client_authentication', 'required'
+  setting 'xpack.security.transport.ssl.verification_mode', 'certificate'
+
+  // It is intentionally to use none for both verification_mode and client_authentication.
+  // Because SSL is not wired up properly on the client (QC) side. These settings
+  // just test that they can be configured.
+  // Once SSL is all wired up, we will need (1) proper SSL verification and
+  // (2) different set of key and cert than the ones used for the transport interface.
+  setting 'xpack.security.remote_cluster.ssl.key', 'transport.key'
+  setting 'xpack.security.remote_cluster.ssl.certificate', 'transport.cert'
+  setting 'xpack.security.remote_cluster.ssl.verification_mode', 'none'
+  setting 'xpack.security.remote_cluster.ssl.client_authentication', 'none'
```

Comment:
This section is basically a test for the PR's change, i.e. the SSL settings for remote access is prefixed with `xpack.security.`.